shoop/front/basket/update_methods.py

```diff
@@ -14,6 +14,12 @@
 from shoop.utils.numbers import parse_decimal_string
 
 
+def get_error_text(error):
+    if hasattr(error, "__iter__"):
```

Comment:
Trying to check if the error is iterable. We don't want to show errors like ["error 1", "error 2"] I understood that this isn't always the case so added the check. I will add docstring and tests for this function. Also I will rethink if this is even a good place for this function.